src/mainboard/amd/romecrb/src/main.rs

```diff
@@ -262,6 +363,9 @@ pub extern "C" fn _start(fdt_address: usize) -> ! {
         msrs(w);
     }
     p[0] = p[0] + 1;
+
+    amd_init(w);
```

Comment:
Unwrap here too.

Reply:
If it's OK I'll keep those for the time being because I think this thing probably will move to mainboard.rs anyway. I'll bring it up the move tomorrow or Friday (main.rs is already very cluttered with weird specific stuff. Not sure that that's good. On the other hand, being able to `write` is nice)

Reply:
Even better would be to actually process the error message here. (But rust always mags at me that I should handle the Err case even when I do handle it using `match`. I'll try again...)

Reply:
I just `write` the detailed message inside the function `amd_init` now--works fine.